src/core/textures/BaseTexture.js

```diff
@@ -195,10 +214,79 @@ BaseTexture.prototype.update = function ()
 
     this.isPowerOfTwo = bitTwiddle.isPow2(this.realWidth) && bitTwiddle.isPow2(this.realHeight);
 
+    if (this.isSVG()) {
+        this.updateSVG();
+    }
+
     this.emit('update', this);
 };
 
 /**
+ * Helper to check if this texture is loaded from svg.
+ * @return {Boolean} True if source is an svg image
+ */
+BaseTexture.prototype.isSVG = function () {
+    return this.source.type && this.source.type === 'svg';
+};
+
+/**
+ * Updates texture from SVG image using canvas. The original SVG Image is stored
+ * as `origSource` and the created canvas is is now `source`. The SVG is scaled
+ * using `sourceScale`.
+ */
+BaseTexture.prototype.updateSVG = function ()
+{
+    if (this.source.type !== 'svg') {
+        // Do nothing if source is not svg
+        return;
+    }
+
+    if (!this.realWidth || !this.realHeight || this.realWidth <= 1 || this.realHeight <= 1 ||
+        (this.realWidth === 150 && this.realHeight === 150)) {
```

Comment:
I can investigate this a bit further but the source seems to have no info of svg size properties if they were missing. Chrome set it to 150x150 and Firefox to 0x0.

About the power of two, I set the canvas to be power of two and draw the correctly sized image into middle of it. I think I had some problems with non-pow2 canvases. I can check that too why I ended up with that solution.